src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -228,18 +230,36 @@ class adminLogs extends viewModelBase {
         return item.includes("EXCEPTION:") || item.includes("Exception:") || item.includes("FATAL ERROR:");
     }
     
-    private isCustomizedItem(item: string): boolean {
-        return (item.includes("Connection established") || item.includes("Connection closed") || item.includes("Connection paused"));
+    private isStudioItem(item: string): boolean {
+        return item.includes(adminLogs.studioMsgPart);
     }
     
+    private getAddedClass(item: string) {
+        const isStudioItem = this.isStudioItem(item);
```

Comment:
move this assigment after first if - no need to calculate that if this is an error.

Reply:
done